assets/stories/skate-components/pages/ladderPage.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import type { Meta, StoryObj } from "@storybook/react"
+import React from "react"
+
+import LadderPage from "../../../src/components/ladderPage"
+import { RoutesProvider } from "../../../src/contexts/routesContext"
+import { LadderDirection } from "../../../src/models/ladderDirection"
+
+import stateFactory from "../../../tests/factories/applicationState"
+import routeFactory from "../../../tests/factories/route"
+import routeTabFactory from "../../../tests/factories/routeTab"
+import { RealDispatchWrapper } from "../../../tests/testHelpers/wrappers"
+
+routeFactory.rewindSequence()
+const routes = routeFactory.buildList(6, { garages: ["Garage A", "Garage B"] })
```

Comment:
**nitpick:** the `routeFactory` by default creates routes with names like `Route 1`, which has been fine for our unit tests and serves to distinguish the name from the ID. However, for visual purposes these look very unlike our actual route designations. What do you think about updating it so that the route names look more "realistic?"

Reply:
I was half expecting the change to end up affecting a bunch of tests, but it worked out of the box!
I like how descriptive it is, but I'm good either way.

Should we just use the sequence number? or randomly add extra tokens, like `16L` or `443/444` etc.

Reply:
f2eeaf87fb153c3e90888de836231f37c0a28c7d

Reply:
Oh, sorry, I meant to respond to your above comment when I first saw it! Yeah, I think anything that results in unique strings of somewhere between 1-3 numerals is pretty realistic for our use case here.